keyboards/montex/readme.md

```diff
@@ -0,0 +1,15 @@
+# IDOBAO MONTEX
+
+A hotswap numpad from IDOBAO.
+    
+## Support:
+
+* Keyboard Maintainer: [NachoxMacho](https://github.com/NachoxMacho)
+* Hardware Supported: IDOBAO MONTEX
+* Hardware Availability: [IDOBAO](https://www.idobao.net/products/ldobao-montex-pad-hot-swappable-mechanical-keyboard-kit)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add information on how to jump to bootloader, to flash new firmware?

Reply:
I've added basic instructions for accessing the bootloader using this firmware/stock option.